query-node/mappings/src/workingGroups.ts

```diff
@@ -648,6 +652,50 @@ export async function workingGroups_StatusTextChanged({ store, event }: EventCon
   await store.save<StatusTextChangedEvent>(statusTextChangedEvent)
 }
 
+export async function workingGroups_LeadRemarked({ store, event }: EventContext & StoreContext): Promise<void> {
+  const [metadataByte] = new WorkingGroups.LeadRemarkedEvent(event).params
+  const group = await getWorkingGroup(store, event)
+
+  const metadata = deserializeMetadata(RemarkMetadataAction, metadataByte)
+  if (metadata?.moderatePost) {
+    if (group.name !== 'forumWorkingGroup') {
+      return invalidMetadata(`The ${group.name} is incompatible with the remarked moderatePost`)
+    }
+    const { postId, rationale } = metadata?.moderatePost ?? {}
+    if (!postId) {
+      return invalidMetadata('No valid forum post id was provided')
+    }
```

Comment:
This validation is not needed anymore, since in protobuf schema, the post_id field is now required (which means validation will be done by the protobuf serialization logic).